docs/standalone/building-on-electrumsv/rest-api.rst

```diff
@@ -75,47 +172,63 @@ Get account balance (confirmed, unconfirmed, unmatured) in satoshis.
 :Endpoint: ``http://127.0.0.1:9999/v1/{network}/dapp/wallets/{wallet_name}/{account_id}/balance``
 :Regtest example: ``http://127.0.0.1:9999/v1/regtest/dapp/wallets/worker1.sqlite/1/utxos/balance``
 
-remove
-**********************
-Removes transactions in the 'Signed' state.
-
-Deleting transactions in the
-'Dispatched', 'Cleared', 'Settled' states could cause issues and so is
-not supported at this time (a DisabledFeatureError will be returned). If you
-require this feature, please make contact via the Atlantis Slack or the
-MetanetICU slack.
-
-:Method: POST
-:Content-Type: application/json
-:Endpoint: ``http://127.0.0.1:9999/v1/{network}/dapp/wallets/{wallet_name}/{account_id}/txs/remove``
-
-**Request Body Payload**
+**Sample Response**
 
 .. code-block::
 
     {
-        "txids": [<txid1>, <txid2>, ...]    (optional field)
+        "confirmed_balance": 14999694400,
+        "unconfirmed_balance": 98000,
+        "unmatured_balance": 0
     }
 
+remove
```

Comment:
I think that the canonical rest way to do this is to have just /txs with DELETE method to be kind of like noun and verb. Doing /txs/remove would duplicate the DELETE method.

Reply:
That's no problem.